README.md

```diff
@@ -1,7 +1,12 @@
-# Language Independent Interface Types For OpenTelemetry
+# OpenTelemetry Protocol (OTLP) Specification
 
 [![Build Check](https://github.com/open-telemetry/opentelemetry-proto/workflows/Build%20Check/badge.svg?branch=main)](https://github.com/open-telemetry/opentelemetry-proto/actions?query=workflow%3A%22Build+Check%22+branch%3Amain)
 
+This repository contains the [OTLP protocol specification](specification/otlp.md)
+and the corresponding Language Independent Interface Types ([.proto files](opentelemetry/proto)).
+
+## Language Independent Interface Types
```

Comment:
Do the other header entries need to be nested under this? They're currently siblings of "Language Independent Interface Types" rather than children.

Reply:
No, I don't think they need to be nested, there is parent-child relationship here.